src/main/kotlin/net/maizegenetics/phgv2/pathing/AlignmentUtils.kt

```diff
@@ -23,280 +28,457 @@ data class KeyFileData(val sampleName: String, val file1: String, val file2: Str
 
 private val myLogger = LogManager.getLogger("net.maizegenetics.phgv2.utils.AlignmentUtils")
 
+class AlignmentUtils {
```

Comment:
Why a class? And if a class, should it be a singleton?

Reply:
It's a class so I can use the companion object.  This allows us to call the functions like standard Static java  functions(i.e. AlignmentUtils.function()). 

This is from @pjbradbury's suggestion a few weeks ago.